content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,103 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer>
+      Keeping track of two pointers to find a subarray that satisfies some condition.
+      <Quiz.Explanation>
+        Incorrect. The pointers don't necessarily have to represent a subarray.
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Keeping track of two pointers to find a pair of indices that satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
```

Comment:
make a note that the pair of indices correspond to the two pointers or just use "two indices" so that way it's easier to readers to "link"